### ISSUE TITLE
Can you make a improve?

### ISSUE THREAD

**Issue by super999** [paulftw/hiberlite#23]
I want this library could get a single bean by another key ,such as "itemid" or "name" . it seem only one way to get a bean with Database::loadBean function .

**Comment by fnc12** [comment]
Yes. This what you we are really missing. Also it is good to make it a header-only lib.

**Comment by d-led** [comment]
There are currently no resources to add further features (@paulftw, correct?). You might want to want to look at the non-lite alternatives, such as [sqlpp11](https://github.com/rbock/sqlpp11)
